lib/utils/inherits.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const util = require('util');
+const _ = require('lodash');
+
+/** like util.inherits, but also copies over static properties */
+function inherits(constructor, superConstructor) {
```

Comment:
Whoops I thought I removed it from utils.js. Cant use it from there because of cyclic dependency (utils -> data types -> utils.inherits).